drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -94,3 +96,11 @@ config ETH_STM32_CARRIER_CHECK_RX_IDLE_TIMEOUT_MS
 	  PHY's carrier status is re-evaluated.
 
 endif # ETH_STM32_HAL
+
+config ETH_STM32_NOCACHE_MEMORY
```

Comment:
I think this setting should actually be removed. You can just add `select NOCACHE_MEMORY if SOC_SERIES_STM32H7X` in the `ETH_STM32_HAL_ETH`. I initially thought it was a setting specific to this module, but it's an actual Zephyr setting.

Reply:
Changed as suggested.